noxfile.py

```diff
@@ -10,12 +10,17 @@
 
 import os
 import pathlib
+import sys
 import types
-from collections.abc import Sequence
-from typing import Final, Literal, TypeAlias
+from collections.abc import Callable, Sequence
+from unittest import mock
+from typing import Any, Final, Literal, TypeAlias, TypeVar
 
 import nox
 
+with mock.patch("sys.path", [".", *sys.path]):
```

Comment:
?

Reply:
Temporarily hack the `sys.path` to load the companion file `noxfile_utils.py`, since this is run in script mode without a venv. I added a comment.